_docs/applications/mssql.md

```diff
@@ -1,36 +1,40 @@
 ---
 layout: guide
-title: StorageOS Docs - Microsoft SQL Server
+title: Microsoft SQL Server with StorageOS
 anchor: applications
 module: applications/mssql
 ---
 
 
 # ![image](/images/docs/explore/mssqllogo.png) Microsoft SQL Server 2016 with StorageOS
 
- Microsoft's latest SQL Server release represents a major update with availability for Linux platform support as well support to run as a container.
+ Microsoft's latest SQL Server release represents a major update with availability for Linux platform support and support to run as a container.
 
 ## Linux Support
 
-For Ubuntu, the Docker container requires Xenial 16.04.
+Microsoft state SQL Server for Linux runs with any supported Linux distribution.  To date we have only tested with Ubuntu 16.04 which we used for this document and Alpine Linux (which did not work).  Additional requirements include:
+* Minimum of 4 GB of disk space 
+* Minimum of 4 GB of RAM (has been tested to work with 3.25GB)
+
+Starting with SQL Server vNext CTP 1.4, the SQL Server command-line tools are included in the Docker image. If you attach to the image with an interactive command-prompt, you can run the tools locally.
 
 ## More Information
 
-More information on SQL Server for Linux is available on the [Microsoft SQL Server for Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview "Microsoft SQL Server for Linuxy") web page.
+More information on SQL Server for Linux is available on the [Microsoft SQL Server for Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview "Microsoft SQL Server for Linux") web page.
 
-Before you start, please ensure you have StorageOS installed and ready on a Linux cluster - please refer to the *Installation Guide* for further details.
+Before you start, please ensure you have StorageOS installed and ready on a Linux cluster - please refer to the [Cluster install ](../install/clusterinstall.html)section for further details.
 
 ## Install MS SQL Server with StorageOS
 
-1. Get the latest MS SQL Server Linux container up and running
+1. Startup the latest MS SQL Server container on a StorageOS node with a StorageOS persistent volume
```

Comment:
Add a colon to the end of the sentence.

Throughout this doc, Instructions to a user to run a command should end in a colon.